### PR TITLE
PRS in EMML1832 (no.9)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**

### CODE REVIEW

File: new/PRS14302Mamas.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1"></persName>
```
Comment (thea-m) [inline new/PRS14302Mamas.xml:48]: fidal missing

File: new/PRS14298ZaraAbreham.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">ዘርአ፡ አብርሃም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zarʾa ʾAbrǝhām</persName>
+                    <floruit notBefore="1480" notAfter="1550">End of 15th century or first half of 16th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="snap:FatherOf" active="PRS14298ZaraAbreham" passive="PRS14299Faraday"/>
```
Comment (thea-m) [inline new/PRS14298ZaraAbreham.xml:53]: maybe it could be noted that a spiritual relationship is likely? (if that is the case)

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS14298ZaraAbreham.xml (hunk 2 above)
Comment (eu-genia) [reply new/PRS14298ZaraAbreham.xml:53]: I think the relationship is spiritual in which case `snap `is not appropriate

Comment (CarstenHoffmannMarburg) [reply new/PRS14298ZaraAbreham.xml:53]: Do you think it is appropriate to transform it to a dc:relation or would you suggest do omit this information.
E.g.
```
<relation name="dc:relation" active="PRS14298ZaraAbreham" passive="PRS14299Faraday"><desc>He is called the "father"
                        of <persName ref="PRS14299Faraday"/>, what is probably a spiritual relationship.</desc></relation>
```

Comment (thea-m) [reply new/PRS14298ZaraAbreham.xml:53]: I think the dc:relation is fine

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**